app/team.go

```diff
@@ -271,10 +271,10 @@ func (a *App) UpdateTeamMemberRoles(teamId string, userId string, newRoles strin
 	member.SchemeAdmin = false
 
 	for _, roleName := range strings.Fields(newRoles) {
-		role, err := a.GetRoleByName(roleName)
```

Comment:
I guess it was because of shadowing variables, in this case other solution would be declare the role and then use the `=` instead of `:=`, something like:

```go
		var role *model.Role
		role, err = a.GetRoleByName(roleName)
		if err != nil {
			err.StatusCode = http.StatusBadRequest
			return nil, err
		}
```

Reply:
I'll follow your suggestion, it is more elegant